src/test/kotlin/com/atlassian/performance/tools/jvmtasks/api/JitterBackoffTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.atlassian.performance.tools.jvmtasks.api
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import java.time.Duration.ofSeconds
+
+class JitterBackoffTest {
+
+    @Test
+    fun shouldYieldPositiveTime() {
```

Comment:
Isn't it flaky? Do you think it's worth to inject a Random to make the tests more predictable?

Reply:
Good suggestion

Reply:
- [ ] seed it